docs/configuration.md

```diff
@@ -2306,6 +2306,50 @@ showDF(properties, numRows = 200, truncate = FALSE)
   </tr>
 </table>
 
+### Spark Application Garbage Collector
+
+<table class="table">
+  <tr><th>Property Name</th><th>Default</th><th>Meaning</th></tr>
+  <tr>
+    <td><code>spark.cleaner.periodicGC.interval</code></td>
+    <td>30min</td>
+    <td>
+      Controls how often to trigger a garbage collection.
+    </td>
+  </tr>
+  <tr>
+    <td><code>spark.cleaner.referenceTracking</code></td>
+    <td>true</td>
+    <td>
+      Controls whether a ContextCleaner should be created when a SparkContext initializes.
+    </td>
+  </tr>
+  <tr>
+    <td><code>spark.cleaner.referenceTracking.blocking</code></td>
+    <td>true</td>
+    <td>
+      Controls whether the cleaning thread should block on cleanup tasks (other than shuffle, which is controlled by
+      spark.cleaner.referenceTracking.blocking.shuffle Spark property).<br><br>
```

Comment:
You might surround prop names in `<code>` for extra clarity

Reply:
Surrounded.